packages/algob/src/internal/deployer.ts

```diff
@@ -1,4 +1,9 @@
-import type { SSCDeploymentFlags, SSCOptionalFlags, TxParams } from "@algorand-builder/runtime/build/types";
+import { BuilderError } from "@algorand-builder/runtime";
+import type {
+  Account, AccountMap, ASADefs, ASADeploymentFlags,
+  SSCDeploymentFlags, SSCOptionalFlags,
+  TxParams
+} from "@algorand-builder/runtime/build/types";
```

Comment:
Can we import from `src` ?

Reply:
tried but cannot import types

Reply:
I confirm. I think tests are working because we use `ts-node` there. But in examples we use the `tsc` compiler, and the build is only exposing the main file (`index.js`).

Reply:
Could you check if `@algorand-builder/runtime/types` works ? (Since `/src` folder is not built, i think we can do this)

Reply:
already checked, doesn't work, also in htlc-example we are using `build` that's why it is working